### PR TITLE
UI-bug fixes, added create query for MV

### PR DESCRIPTION
**Pull request by sumukhswamy**
### Description
fixed MV load bug
added mv create button
handled error case when post async query fails
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by ps48** [comment]
@sumukhswamy Can you please update the PR title and make it shorter, these title directly go in to the release notes. You can use description for more details.

File: common/constants/index.ts [hunk 1]
```diff
@@ -42,6 +44,18 @@ OPTIONS (
   compression 'gzip'
 );`;
 
+export const CREATE_MATERIALIZED_VIEW = `CREATE MATERIALIZED VIEW datasource.database.index_name
```
Comment (ps48) [inline common/constants/index.ts:47]: Should be `datasource.database.mv_metrics`

File: common/constants/index.ts [hunk 2]
```diff
@@ -21,12 +21,14 @@ export const LOAD_OPENSEARCH_INDICES_QUERY = `SHOW tables LIKE '%';`;
 export const SKIPPING_INDEX_QUERY = `CREATE SKIPPING INDEX ON myS3.logs_db.http_logs 
 (status VALUE_SET) 
 WITH (
-  auto_refresh = true
+  auto_refresh = true,
+  checkpoint_location = 's3://test/'
   )`;
 export const COVERING_INDEX_QUERY = `CREATE INDEX covering_idx ON myS3.logs_db.http_logs
```
Comment (ps48) [inline common/constants/index.ts:27]: Let's change `myS3.logs_db.http_logs` to `datasource.database.table`

File: common/constants/index.ts [hunk 3]
```diff
@@ -21,12 +21,14 @@ export const LOAD_OPENSEARCH_INDICES_QUERY = `SHOW tables LIKE '%';`;
 export const SKIPPING_INDEX_QUERY = `CREATE SKIPPING INDEX ON myS3.logs_db.http_logs
```
Comment (ps48) [inline common/constants/index.ts:21]: Let's change `myS3.logs_db.http_logs` to `datasource.database.table`

File: common/constants/index.ts [hunk 4]
```diff
@@ -21,12 +21,14 @@ export const LOAD_OPENSEARCH_INDICES_QUERY = `SHOW tables LIKE '%';`;
...
   )`;
 export const COVERING_INDEX_QUERY = `CREATE INDEX covering_idx ON myS3.logs_db.http_logs
  (status) 
  WITH (
-  auto_refresh = true
+  auto_refresh = true,
+  checkpoint_location = 's3://test/'
   )`;
 export const CREATE_DATABASE_QUERY = `CREATE DATABASE myS3.logs_db`;
 export const CREATE_TABLE_QUERY = `CREATE EXTERNAL TABLE myS3.logs_db.logs (
```
Comment (ps48) [inline common/constants/index.ts:34]: same for these let's change `myS3.logs_db.http_logs` to `datasource.database.table`

File: common/constants/index.ts [hunk 5]
```diff
@@ -42,6 +44,18 @@ OPTIONS (
   compression 'gzip'
 );`;
 
+export const CREATE_MATERIALIZED_VIEW = `CREATE MATERIALIZED VIEW datasource.database.index_name
+AS SELECT
+   count(field)
+FROM datasource.database.table
+GROUP BY TUMBLE (timestamp, '2 hours')
+ WITH (
+index_settings = '{"number_of_shards":5,"number_of_replicas":3}',
```
Comment (ps48) [inline common/constants/index.ts:53]: can remove index settings option

File: public/components/SQLPage/table_view.tsx [hunk 6]
```diff
@@ -461,8 +580,8 @@ export const TableView = ({ http, selectedItems, updateSQLQueries, refreshTree }
       label: createLabel(table, database.name, index),
       id: `${database.name}_${table.name}`,
       icon:
-        table.type === TREE_ITEM_LOAD_MATERIALIZED_BADGE_NAME ? null : table.type ===
-          TREE_ITEM_BADGE_NAME ? null : (
+        table.type === TREE_ITEM_MATERIALIZED_VIEW_DEFAULT_NAME ? <EuiBadge>MV</EuiBadge> : table.type ===
```
Comment (ps48) [inline public/components/SQLPage/table_view.tsx:583]: minor: can move creation to a function.

**Review by ps48 — APPROVED** [review]
minor changes

**Review by mengweieric — APPROVED** [review]

**Review by ps48 — APPROVED** [review]

**Pull request merged**